fspiop/v1_0/openapi3/components/schemas/Refund.yaml

```diff
@@ -0,0 +1,14 @@
+title: Refund
+type: object
+description: Data model for the complex type Refund.
+properties:
+  originalTransactionId:
+    type: string
+    description: Reference to the original transaction ID that is requested to be refunded.
+    example: b51ec534-ee48-4575-b6a9-ead2955b8069
```

Comment:
as an example we are using the CorrelationId value (uuid v4)- but this type isn't used here - strange inconsequency...